samples/ControlGallery/ControlGallery/Views/SkiaCanvasDemo.razor

```diff
@@ -0,0 +1,13 @@
+﻿@using Microsoft.MobileBlazorBindings.SkiaSharp
+@page "/skiacanvas/"
```

Comment:
Would it be "nice" to use hierarchical URLs? Does it work if you do like `/skia/canvas` etc. for all the pages? Otherwise some of the URLs later in the PR look like it's a keyboard mash `/skialasflahdgkahegauh` 😄

Reply:
If I'm not going to use hierarchical URL, why did i bother building them? Fixed.